projects/ardin_mb/state_handler.cc

```diff
@@ -55,7 +73,7 @@ bool StateHandler::handleEvent(State state, Event event)
     if(event == Event::Update) {
 
         // Render top down and ants eye view
-        m_Renderer.renderPanoramicView(m_Pose.y(), m_Pose.x(), 0.01_m,
+        m_Renderer.renderPanoramicView(m_Pose.x(), m_Pose.y(), m_PathHeight,
```

Comment:
Out of interest, why were the x and y inverted before and why not now?

Reply:
Nvm -- I just noticed that you said why in the PR (which I probably should have read before the code review rather than after :confused: )

Reply:
I had a quick look through the history and exactly what happened here isn't clear but still...